common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -284,7 +293,6 @@ static MergedShuffleFileManager newMergedShuffleFileManagerInstance(TransportCon
       // will also need the transport configuration.
       return mergeManagerSubClazz.getConstructor(TransportConf.class).newInstance(conf);
     } catch (Exception e) {
-      logger.error("Unable to create an instance of {}", mergeManagerImplClassName);
```

Comment:
does something else print a useful error message?

Reply:
Cannot use the logger inside this static context. Therefore, ended up removing it.

Reply:
I agree with @tgravescs, dropping the log message (particularly an error) would be missing out on very useful debugging information.
One option would be to move it out of this class into some other util ?

Reply:
Perhaps we need to create a static logger (w/o prefix) to fall back to, in the case that we're unable to access the prefixed instance logger?